.gitignore

```diff
@@ -41,3 +41,6 @@ credentials
 
 #issues database
 issues.db
+
+#yarn
+yarn.lock
```

Comment:
Specifically, https://github.com/yarnpkg/yarn/issues/1583#issuecomment-294062586 (primarily) & https://github.com/yarnpkg/yarn/issues/1583#issuecomment-279409311, which each say https://github.com/yarnpkg/yarn/issues/1583#issuecomment-257544217 is wrong under different circumstances.